nevergrad/functions/control/core.py

```diff
@@ -88,25 +121,63 @@ def evaluation_function(self, *recommendations: p.Parameter) -> float:
 
 class Ant(BaseFunction):
     env_name = 'Ant-v2'
-    policy_dim = (8, 111)
+    policy_dim: tp.Tuple[int, ...] = (111, 8)
```

Comment:
how come this has changed shape? I'm afraid this could actually change how state mean is processed, but I am not very familiar with this so I may be wrong
@teytaud maybe this needs cross-checking

Reply:
I changed the order to match (input layer, ..., output layer) for the sake of clarity. It shouldn't interfere with state_mean because we normalize only the input of the first layer. But cross-checking always helps :)

Reply:
@teytaud how can we make sure this works as expected?

Reply:
Can you define "as expected" ? <<parametrization: p.Tuple = p.Tuple(*list_parametrizations).set_name(self.env_name)>> is perfect to me, in the sense "this is a tuple with one array per layer". 

In terms of impact with the state mean: this is unrelated, the state mean and std are in the domain of the control, not in the domain of the weights.

Reply:
(we might consider other parametrizations, in particular for progressive methods, but this will be for another PR...)